typings/@chainlink/index.d.ts

```diff
@@ -82,6 +82,24 @@ declare module '@chainlink/types' {
     chain?: ChainType
     balance?: number
   }
+
+  type DNSResponseQuestion = {
+    name: string
+    type: number
+  }
+
+  type DNSResponseAnswer = {
+    name: string
+    type: number
+    TTL: number
+    data: string
+  }
+
+  export type DNSResponse = {
+    Status: number
+    Question: DNSResponseQuestion[]
+    Answer: DNSResponseAnswer[]
+  }
```

Comment:
Please move these types to the `dns-query` adapter, as they are specific for that domain.

Our DNS base adapter should be the source of all DNS domain types.

Reply:
This `typings/@chainlink` was a bucket for all types because our core packages were not written in TS. Now that we migrated those libs, we can start moving those types there.